data/421/182/331/421182331.geojson

```diff
@@ -17,6 +17,12 @@
     "mz:is_current":-1,
     "mz:min_zoom":12.0,
     "mz:note":"quattroshapes points import (201603)",
+    "name:ara_x_preferred":[
+        "\u0645\u064f\u062d\u0627\u0641\u0638\u0629 \u0637\u0631\u0637\u0648\u0633"
+    ],
+    "name:eng_x_preferred":[
+        "Tartus Governorate"
```

Comment:
This is a locality, so maybe just `Tartus` as a capital city name???

Reply:
Yeah, reviewing the English name translations, it looks like there are other cases of placetypes being appended to names (especially in Japan).

I'm going to rework the English names a bit and add a commit to each PR with updated name properties and new label properties when needed.

Reply:
fixed in d9821df